source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -132,33 +131,17 @@ A Client MUST use options specified in the Connection String, and options
 passed in as parameters in code to the MongoClient constructor (or equivalent
 API for each driver), to override options provided through TXT records.
 
-If listable options, such as ``readPreferenceTags``, are present in a TXT
-record AND in MongoClient constructor (URI, or options passed in code), then a
-Client MUST discard *all* elements of this listable option obtained through
-the TXT record(s) and *only* use these elements from the MongoClient
-constructor.
-
 .. _`Connection String spec`: ../connection-string/connection-string-spec.rst#defining-connection-options
 
 If any connection string option in a TXT record is incorrectly formatted, a
-Client must throw a parse exception.
-
-Authentication credentials MUST NOT be supported as options through TXT
-records.
+Client MUST throw a parse exception.
 
 This specification does not change the behaviour of handling unknown keys or
```

Comment:
This now seems contradictory to the new text above which says:

> and MUST raise an error if any other option is encountered

So now, an incorrect value for either of the two supported options should be treated the same, but unknown keys are not permitted.

Reply:
I see you've already discussed this above.  I'd like to see a change as well.